packages/riipen-ui/src/components/Popover.jsx

```diff
@@ -126,7 +156,7 @@ class Popover extends React.Component {
   };
 
   getPositioningStyle = () => {
```

Comment:
Maybe rename this method to `setPosition` or `updatePosition` or `setPositionStyle` etc.etc. something with set or update now that it is setting / updating.

Reply:
👍